src/util/constant.js

```diff
@@ -0,0 +1,22 @@
+export const ROUTE = {
+  랜딩: "/",
+  로그인: "/signin",
+  개인정보처리방침: "/privacy",
+  FAQ: "/faq",
+};
+
+const second = 1000;
+const minute = second * 60;
+const hour = minute * 60;
+const day = hour * 24;
+const month = day * 31;
+const year = month * 12;
```

Comment:
constants는 보편적으로 대문자로 변수명을 짓습니다 ! 다음과 같이 작성해보면 어떨까요 ? =)
```suggestion
const SECOND = 1000;
const MINUTE = SECOND * 60;
const HOUR = MINUTE * 60;
const DAY = HOUR * 24;
const MONTH = DAY * 31;
const YEAR = MONTH * 12;
```